app/interactors/tags/update_interactor.rb

```diff
@@ -6,13 +6,16 @@ class Tags::UpdateInteractor
   delegate :params,
            :user,
            :edition,
+           :issues,
            to: :context
 
   def call
     Edition.transaction do
       find_and_lock_edition
       update_edition
 
+      check_for_issues
+
```

Comment:
I remember ordering it this way for a reason initially, specifically that there were no issues reported when update occurred first. However, this may not be needed anymore due to changes since, I'll have a look.